store/imagestore/remote.go

```diff
@@ -19,9 +19,14 @@ package imagestore
 
 import (
 	"database/sql"
+	"errors"
 	"time"
 )
 
+var (
+	ErrRemoteNotFound = errors.New("remote with the given ACI URL not found")
```

Comment:
Given that we are already planning to integrate non-ACI stores, can we remove ACI references from there? A generic `remote target not found` should suffice.